src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,76 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
The `movie` field is annotated with `@OneToOne`, which implies a one-to-one relationship between `MovieSession` and `Movie`. If a movie can have multiple sessions, consider using `@ManyToOne` instead to accurately represent the relationship.